app/javascript/packages/document-capture/components/in-person-outage-alert.tsx

```diff
@@ -0,0 +1,34 @@
+import { t } from '@18f/identity-i18n';
+import { Alert } from '@18f/identity-components';
+import { useContext } from 'react';
+import { getConfigValue } from '@18f/identity-config';
+import { InPersonContext } from '../context';
+
+function InPersonOutageAlert() {
+  const { inPersonOutageExpectedUpdateDate } = useContext(InPersonContext);
+
+  return (
+    <Alert type="warning" className="margin-bottom-4" textTag="div">
+      <>
+        <p className="margin-bottom-2">
+          <strong>
+            {t('idv.failure.exceptions.in_person_outage_error_message.post_cta.title', {
+              date: new Intl.DateTimeFormat(document.documentElement.lang, {
+                weekday: 'long',
+                month: 'long',
+                day: 'numeric',
+              }).format(new Date(inPersonOutageExpectedUpdateDate)),
```

Comment:
this displays the formatted date as expected but the new Date() is an invalid date and the spec fails. Any thoughts on how I can create a new valid date using a given string, for ex `october 31, 2020`

Reply:
If this value is coming from the server, I think it'd be best to pass it as an ISO-8601 date string and avoid formatting it to a human-readable string until it's actually shown.

There might be a couple concerns with trying to parse the human-readable string:

- ISO 8601 is the only official format required to be supported for string parsing, so it's possible it wouldn't work consistently across browsers
- It could get weird with timezone handling (UTC vs. the user's local timezone)

Resources:

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#date_time_string_format
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse#non-standard_date_strings